src/components/common/ValidateForm/ValidateForm.tsx

```diff
@@ -105,7 +105,7 @@ function ValidateFailure({ formId, validationError, onClickValidate, validatedTi
 export function ValidateForm({ id, onClickValidate, validationError, isChartRepo = false, validationStatus = "", configName, warning = "" }) {
     return (
         <div className="mt-16">
-            {!id && configName === 'chart repo' && validationStatus != VALIDATION_STATUS.LOADER &&
+            {!id && configName !== 'chart repo' && configName !== 'gitops ' && validationStatus != VALIDATION_STATUS.LOADER &&
```

Comment:
Is reversal of check `configName !== 'chart repo'` intentional?

Reply:
Yes it is itentional, since we are not showing the info icon and the message anymore, but maybe used by other components in the future thats why I have reversed the check, so that if the validate form comes from chart repo then the info icon will not show.